### ISSUE TITLE
MongoParseError: Multiple text records not allowed

### ISSUE THREAD

**Issue by TommysG** [Automattic/mongoose#9489]
**Do you want to request a *feature* or report a *bug*?**

Report a **bug**
**What is the current behavior?**

Randomly after i make some changes into the code and rerun with nodemon it shows me this error. But if i restart it multiple times, it works fine again and connects to my db:  

`MongoParseError: Multiple text records not allowed
    at QueryReqWrap.callback (C:\Users\acer\Desktop\thomas\websites\food-app\node_modules\mongodb\lib\core\uri_parser.js:97:27)
    at QueryReqWrap.onresolve [as oncomplete] (dns.js:205:10)`

**If the current behavior is a bug, please provide the steps to reproduce.**

i use the default URI that recommends MongoDB Atlas on how to connect to your cluster.
I also tried many URIs from mongoose docs, but none of them really worked.

```js

// ============> CONNECT TO MONGODB
const mongoURI = `mongodb+srv://admin:${process.env.PASS}@cluster0.fewzf.mongodb.net/${process.env.DBNAME}?retryWrites=true&w=majority`;

mongoose
  .connect(mongoURI, {
    useCreateIndex: true,
    useNewUrlParser: true,
    useUnifiedTopology: true,
  })
  .catch((err) => console.log(err));

mongoose.connection.once("open", () => {
  console.log("DB CONNECTED");
})
```

**What is the expected behavior?**
to connect to mongodb everytime without showing this error: `MongoParseError: Multiple text records not allowed`

**What are the versions of Node.js, Mongoose and MongoDB you are using? Note that "latest" is not a version.**
```
Node.js: 12.9.0
Mongoose: 5.10.9
MongoDB: atlas
```

**Comment by vkarpov15** [comment]
Use the non `+srv` MongoDB connection string. Instead of using the v3.6+ connection string, use the v2.2+ connection string:

![image](https://user-images.githubusercontent.com/1620265/99720928-74462f00-2a7c-11eb-942d-a68691a0db5b.png)

It should look like `mongodb://admin:${process.env.PASS}@cluster0-1.fewzf.mongodb.net:27017,cluster0-2.fewzf.mongodb.net:27017,cluster0-3.fewzf.mongodb.net:27017`.

This issue is due to [Node's `dns.resolveTxt()` function](https://nodejs.org/api/dns.html#dns_dns_resolvetxt_hostname_callback) returning a result that MongoDB can't parse, which is indicative of DNS issues. We've seen some ISPs block `+srv` URIs altogether.

TLDR; if you're running into this issue locally, use the non `+srv` connection string. We still recommend using `+srv` in production, but it can be a convenient workaround for network issues.
